### ISSUE TITLE
[1.0.0.18][Minor] Landing a dragonboost in-between two top-side loop pieces provides recoil between them

### ISSUE THREAD

**Issue by benjistring** [whitelilydragon/ShangMuArchitect#135]
https://user-images.githubusercontent.com/69790793/108465373-bd05b380-7236-11eb-9faf-b9ae161922c8.mp4

**Comment by benjistring** [comment]
Similar issue with Milla's flight inbetween them!


https://user-images.githubusercontent.com/69790793/108477460-16c2a980-7248-11eb-9f51-3ccec43eee9b.mp4

**Comment by whitelilydragon** [comment]
I compared this to Freedom Planet, and seemingly, Lilac can clip to top-left or top-right inside slopes *regardless of angle*, for as long as she's dragon-boosting, so I've fixed that!

And I did what I could to make Milla's flutter more consistent, as well. (She already had conditional checks against flutter clipping to ceilings elsewhere, but not in the slope ceiling sensor!) But it may still act funny if the player was super persistent, I'm sorry; can't really do much about that.
